tests/test_gcloud.py

```diff
@@ -38,9 +38,9 @@ def test_open_read(self):
 
         f = self.storage.open(self.filename)
         self.storage._client.bucket.assert_called_with(self.bucket_name)
-        self.storage._bucket.get_blob.assert_called_with(self.filename)
```

Comment:
maybe define the `DEFAULT_TIMEOUT=60` and use it instead of the `60` in the test. It will make the tests easier to update if the default is ever changed. It also make it very explicit what is going on.